timescaledb/getting-started/create-cagg/create-cagg-basics.md

```diff
@@ -1,256 +1,91 @@
-to create:
+Now that you have the aggregation query, you can use it to create a continuous aggregate. 
 
-* 6-month aggregates for all metrics for a specific city
-* 30-day aggregates for temperature and rainfall for all cities
-* 5-year aggregates for all metrics for certain cities
+The `CREATE MATERIALIZED VIEW` triggers the database to create a materialized view with 
+the name specified, in this case being `stock_candlestick_daily`. In the next line, 
+`WITH (timescaledb.continuous)` lets the database know that you want to create a continuous 
+aggregate and not just a generic materialized view. Then, the `AS` keyword is needed to specify 
+the aggregate query you want to use for creating the continuous aggregate. 
 
-<highlight type="tip">
-If you ever need to inspect details about a continuous aggregate, such as its 
-configuration or the query used to define it, you can use the following 
-informational view:
-</highlight>
+To do this, use the following code:
 
 ```sql
--- See info about continuous aggregates
-SELECT * FROM timescaledb_information.continuous_aggregates;
+CREATE MATERIALIZED VIEW stock_candlestick_daily
+WITH (timescaledb.continuous) AS
+SELECT
+	time_bucket('1 day', "time") AS day,
+	symbol,
+	max(price) AS high,
+	first(price, time) AS open,
+	last(price, time) AS close,
+	min(price) AS low
+FROM stocks_real_time srt
+GROUP BY day, symbol;
 ```
 
-### Step 2: Populate the continuous aggregate
-
-You've created the continuous aggregate but it has not materialized any data.
-There are two ways to populate a continuous aggregate: via manual refresh or an
-automation policy. These methods enable you to refresh materialized data in your
-continuous aggregates when it's most convenient for you (for example, during low
-query-load times on your database).
-
-Let's take a look at how to do each one.
-
-#### Manual refresh
+By default, this code will create the aggregate **and** materialized the aggregated data. 
 
-You can manually refresh a continuous aggregate using a "one-shot refresh."
-This is useful for refreshing data for only a specific time period in the past
-or if you want to materialize a lot of data at once on a once-off basis.
-
-The example below refreshes the continuous aggregate `weather_metrics_daily` for
-the time period starting January 1, 2010 and ending January 1, 2021:
-
-```sql
--- manual refresh
--- refresh data between 1 Jan 2010 and 2021
-CALL refresh_continuous_aggregate('weather_metrics_daily','2010-01-01', '2021-01-01');
-```
+  <img class="main-content__illustration" src="https://s3.amazonaws.com/assets.timescale.com/docs/images/getting-started/continuous-aggregate.jpg" alt="Continuous aggregate upon creation"/>
 
-Querying our continuous aggregate for data older than January 1, 2009 shows that
-it is indeed populated with data from the above time frame, as the first row
-returned is for January 1, 2010:
+This is why the query may take some time to run. From this point on, query the data from the 
```

Comment:
```suggestion
The query may take some time to run because it needs to perform these calculations. 
After the calculation results are stored, querying the data from the 
```